src/jsonLogic.js

```diff
@@ -399,6 +399,7 @@ function checkRuleIntegrity(
     subRule.map((item) => {
       const isVar = item !== null && typeof item === 'object' && Object.hasOwn(item, 'var');
       if (isVar) {
+        if (Array.isArray(item.var)) return; // Is an accumlator for reduce and can't be checked.
```

Comment:
**Suggestion**

Just wondering if there could ever be an array path which we would want to validate? 🤔 
Might be safer to be less permissive: `const isReducerVar = operator === 'reduce' && Array.isArray(item.var)`
This also makes the code more explicit so we don't need code comments. 🙂